muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/evaluation/EvaluationJsPluginMessages_fi.properties

```diff
@@ -1,4 +1,44 @@
 plugin.evaluation.noWorkspaces = Ei ty�tiloja
+plugin.evaluation.notifications.gradesystem.error = Virhe ladattaessa arviointiasteikoita. P\u00E4ivit\u00E4 sivu uudelleen ja yrit\u00E4 uudestaan.
```

Comment:
This unicode noise in this file is not a problem?

Reply:
No, works just fine. 

However it could be worth while to examine where (and why) those unicodes are coming from. This might be happening when editing locale strings outside of eclipse which should not be a problem.